### PR TITLE
Added code styling

### PR DESCRIPTION
**Pull request by CaioHVectorA**

### CODE REVIEW

**Comment by CaioHVectorA** [comment]
#2

**Comment by CaioHVectorA** [comment]
By the way: i also added a code to add to each h2 title the id with their text, and it made possible to add summarys in HTML too

File: src/components/article.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { useEffect } from "react";
+import { highlight } from "sugar-high";
+
+export function Article({ html }: { html: string }) {
+    const handleFormatCode = () => {
+        const codes = document.querySelectorAll("pre code");
+        codes.forEach((code) => {
+        if (code.innerHTML.includes("span")) return;
+          const html = highlight(code.textContent as string);
+          // console.log(html)
```
Comment (LukeberryPi) [inline src/components/article.tsx]:
```suggestion
          
```

**Comment by LukeberryPi** [comment]
great addition!

**Pull request merged**